scout/adapter/mongo/event.py

```diff
@@ -184,9 +184,9 @@ def case_events_by_verb(self, category, institute, case, verb):
         }
         return self.event_collection.find(query)
 
-    def user_events(self, user_obj=None):
+    def user_events(self, user_obj=None, case=None):
```

Comment:
Hm, but querying for` case=None` is probably not what you want? Rather perhaps different queries if the case param is given.